recipe/meta.yaml

```diff
@@ -56,16 +69,18 @@ test:
     - if not exist %PREFIX%\\Library\\bin\\libtiffxx.dll exit 1  # [win]
 
 about:
-  home: https://libtiff.gitlab.io/libtiff/
-  license: libtiff
+  # HTTPS does not work.
+  home: http://www.simplesystems.org/libtiff/
+  license: HPND
   license_file: COPYRIGHT
-  license_family: OTHER
-  summary: 'Support for the Tag Image File Format (TIFF).'
+  summary: Support for the Tag Image File Format (TIFF).
   description: |
     This software provides support for the Tag Image File Format (TIFF), a
     widely used format for storing image data.
-  dev_url: https://gitlab.com/libtiff/libtiff
+  # HTTPS does not work.
   doc_url: http://www.simplesystems.org/libtiff/
+  doc_source_url: https://gitlab.com/libtiff/libtiff/-/blob/master/README.md
```

Comment:
`https://gitlab.com/libtiff/libtiff/blob/master/README.md` goes to the same page and looks cleaner.